src/test/rpc/AccountInfo_test.cpp

```diff
@@ -291,11 +292,82 @@ class AccountInfo_test : public beast::unit_test::suite
         }
     }
 
+    void testBatch()
+    {
+        using namespace jtx;
+        Env env(*this, features(featureMultiSign));
+        Account const alice {"alice"};
+        env.fund(XRP(1000), alice);
+
+        auto const withoutSigners = std::string ("[{ ") +
```

Comment:
This would probably read a little better as a raw literal  (`R"`), or just as a `Json::Value` with single field assigments.